src/test/java/edu/cornell/kfs/concur/service/impl/ConcurEventNotificationServiceImplTest.java

```diff
@@ -27,5 +29,43 @@ public void validateFindingNotificationId() {
         String actualNotificationId = concurEventNotificationServiceImpl.findNotificationId(notificationURI);
         assertEquals("Expected NotificationId and ActualNotificationId should be the same", expectedNotifcationId, actualNotificationId);
     }
+    
+    @Test
+    public void validateShortRespone() {
+        String shortMessage = "some short message";
+        String actualResults = concurEventNotificationServiceImpl.truncateValidationResultMessageToMaximumDatabaseFieldSize(shortMessage);
+        assertEquals("short message should be the same", shortMessage, actualResults);
+    }
+    
+    @Test
+    public void validateNullRespone() {
+        String nullMessage = null;
+        String actualResults = concurEventNotificationServiceImpl.truncateValidationResultMessageToMaximumDatabaseFieldSize(nullMessage);
+        assertEquals("null message should be the same", nullMessage, actualResults);
+    }
+    
+    @Test
+    public void validateLongRespone() {
+        String longMessage = buildLongMessage();
+        assertEquals("long message should 2000 characters long", ConcurConstants.VALIDATION_RESULT_MESSAGE_MAX_LENGTH, longMessage.length());
+        String actualResults = concurEventNotificationServiceImpl.truncateValidationResultMessageToMaximumDatabaseFieldSize(longMessage);
+        assertEquals("long message should be the same", longMessage, actualResults);
+    }
+    
+    @Test
+    public void validateOverageRespone() {
+        String longMessage = buildLongMessage();
+        String overageMessage = longMessage + "some extra stuff to exceed max length";
+        String actualResults = concurEventNotificationServiceImpl.truncateValidationResultMessageToMaximumDatabaseFieldSize(overageMessage);
+        assertEquals("message should be the long message without the extra stuff", longMessage, actualResults);
+    }
+    
+    private String buildLongMessage() {
+        StringBuilder sb = new StringBuilder(ConcurConstants.VALIDATION_RESULT_MESSAGE_MAX_LENGTH);
+        for (int i = 0; i<ConcurConstants.VALIDATION_RESULT_MESSAGE_MAX_LENGTH; i++) {
+            sb.append("A");
+        }
+        return sb.toString();
```

Comment:
Since you're just repeating the same value to build the long string, you could simplify this by calling StringUtils.repeat() instead.